frontend/src/components/Buttons/FilledButton/index.tsx

```diff
@@ -28,7 +28,6 @@ export const FilledButton: React.FC<ButtonProps> = ({
                 { 'px-6': !icon },
                 bg(colorPalette),
                 textOnBg(colorPalette),
-                bgHover(colorPalette),
```

Comment:
Change this to
`!disabled && bgHover(colorPalette),`
This will cause hover color to be added in classNames only when the button is not disabled.

Reply:
Yes thanks